#We Got Coders - My Amazing Application.html#

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
```

Comment:
Well, this file has got the most line. So, I don't think, it is a good idea to delete it.